modules/apps/mail/mail-messaging-impl/src/main/java/com/liferay/mail/messaging/internal/MailMessageListener.java

```diff
@@ -136,12 +134,15 @@ protected InternetAddress filterInternetAddress(
 			return null;
 		}
 
-		if (_mailSendBlacklist.contains(emailAddress)) {
+		Set<String> mailSendBlacklist = new HashSet<>(
+			Arrays.asList(_mailSettingSystemConfiguration.mailSendBlacklist()));
+
+		if (mailSendBlacklist.contains(emailAddress)) {
 			if (_log.isWarnEnabled()) {
 				_log.warn(
 					StringBundler.concat(
 						"Email ", emailAddress, " will be ignored because it ",
-						"is included in ", PropsKeys.MAIL_SEND_BLACKLIST));
+						"is included in mail.send.blacklist"));
```

Comment:
@jeffwu0724 The log message does not make sense because you have remove the portal property "mail.send.blacklist"